x/pairing/README.md

```diff
@@ -0,0 +1,402 @@
+
+Every epoch start, the amount of complainers CU is compared with the amount of serviced CU of each provider across a few epochs back. If the complainers CU is higher, the provider is considered unresponsive and gets punished. The number of epochs back is determined by the recommendedEpochNumToCollectPayment parameter
+
+#### Static Providers
+
+Static providers are Lava chain providers that offer services to any consumer without relying on pairing. This feature allows new consumers to communicate with the Lava chain without a centralized provider. For example, when a new consumer wants to start using Lava, it needs to obtain its pairing list from a Lava node. However, since it initially does not have a list of providers to communicate with, it can use the static providers list to obtain its initial pairing list.
+
+To clarify, a consumer cannot randomly find a Lava provider's endpoint and receive service from it because a provider will only serve consumers whose pairing with it can be verified. Therefore, the consumer must obtain the providers' pairing list from a Lava node.
+
+Note that unstaking static providers will have their coins returned after a longer period of time compared to regular ("dynamic") providers due to their special status.
+
+### Payments
+
+The Lava network connects blockchain data providers with consumers who need access to that data. Once a pairing is established, a consumer can send a message to the provider to request a specific blockchain query. The provider then sends a relay request to the blockchain, retrieves the required information, and forwards it to the consumer. After completing the requested service, the provider sends a relay payment transaction to the Lava network to receive payment.
+
+Here is the workflow for Lava's payment mechanism:
+
+1. The provider sends a relay payment transaction, which is signed by the consumer.
+2. The reported CU (compute units) from the relay payment transaction is tracked.
+3. On a monthly basis, all the providers' tracked CU are reviewed, and payments are sent for their accumulated services.
+
+#### CU Tracking
+
+The goal of CU tracking is to keep records of serviced CUs by the providers, in order to determine the amount of payment they should receive at the end of the month. When a relay payment transaction occurs, the number of CUs associated with each relay is counted and saved under the provider who initiated the transaction. At the end of each month, the CU tracker is reset for all providers.
+
+#### Providers Payment
+
+Providers receive payments once a month. These payments are funded by the funds that consumers paid for their subscriptions. The amount of coins to be sent to each provider depends on the number of tracked CUs they had over the previous month. The payment is calculated using the following formula:
```

Comment:
this is not accurate, they recieve payment whenever a subscription expires

Reply:
done. see https://github.com/lavanet/lava/commit/6de4aabf932a0ccf06e72626912782c8de252add